module ApiErrors
  API_ERRORS = {
      VOLUNTEER_NOT_FOUND: :VOLUNTEER_NOT_FOUND,
      INVALID_CAPTCHA: :INVALID_CAPTCHA,
      INVALID_VERIFICATION_CODE: :INVALID_VERIFICATION_CODE,
      INVALID_TOKEN: :INVALID_TOKEN,
      INVALID_ARGUMENT: :INVALID_ARGUMENT,
      UNAUTHORIZED_RESOURCE: :UNAUTHORIZED_RESOURCE,
      REQUEST_CAPACITY_EXCEEDED: :REQUEST_CAPACITY_EXCEEDED,

      UNEXPECTED_ERROR: :UNEXPECTED_ERROR
  }.freeze

  def self.[](value)
    API_ERRORS[value]
  end
end

